README.md

```diff
@@ -4,8 +4,54 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/Peripli/service-manager)](https://goreportcard.com/report/github.com/Peripli/service-manager)
 [![Coverage Status](https://coveralls.io/repos/github/Peripli/service-manager/badge.svg?branch=master)](https://coveralls.io/github/Peripli/service-manager?branch=master)
 
-Service Manager Core Module
+## What is Service Manager
 
-[Run Service Manager on CF](deployment/cf/README.md)
+Service Manager is a central registry for service brokers and platforms registration. It tracks service instances creation and allows sharing of service instances between different Platform Instances.
 
-[Run Service Manager on Kubernetes](deployment/k8s/README.md)
+## Setup Service Manager Components
+
+### Deploy Service Manager
+
+Currently the Service Manager can be deployed on CF or on Kubernetes.
+
+For more information see:
+
+* [Deploy to CF](deployment/cf/README.md)
+* [Deploy to Kubernetes](deployment/k8s/README.md)
+
+After the deployment you need to have the Service Manager url.
+
+For CF/PCF Dev you can get the url with the `cf app <service-manager-app-name>` command. For example, if your *service-manager-app-name* is *service-manager* the command will be:
```

Comment:
> For CF/PCF Dev you can get the address with the following command:
> ```sh
> cf app service-manager
> ```

Reply:
This will not be entirely true if the application name is not service-manager